boards/shields/st_b_lcd40_dsi1_mb1166/Kconfig.defconfig

```diff
@@ -4,18 +4,14 @@
 
 if SHIELD_ST_B_LCD40_DSI1_MB1166 || SHIELD_ST_B_LCD40_DSI1_MB1166_A09
 
-# Double frame buffer maintained by lvgl.
 if LVGL
 
+if BOARD_STM32H747I_DISCO_STM32H747XX_M7
```

Comment:
I'd rather add a `orsource "boards/*.defconfig"` then do this in a `boards/shields/st_b_lcd40_dsi1_mb1166/boards/stm32h747i_disco_stm32h747xx_m7.defconfig`